explainability-service/src/main/java/org/kie/trustyai/service/data/storage/MinioStorage.java

```diff
@@ -203,4 +203,9 @@ public boolean inputExists() throws StorageReadException {
     public boolean outputExists() throws StorageReadException {
         return false;
     }
+
+    @Override
+    public long getLastModified() {
+        return isObjectAvailable(bucketName, inputFilename).lastModified().toInstant().toEpochMilli();
```

Comment:
What `isObjectAvailable` returns if the value is not available? This code seems not managing that case

Reply:
If the object is not available, it will now throw a random value (forcing a cache miss and going through the proper flow of handling missing data files)